crates/aptos-rosetta/src/block.rs

```diff
@@ -151,14 +151,12 @@ impl BlockCache {
         let mut versions = BTreeMap::new();
         // Genesis is always index 0
         // TODO: Ensure that this won't fail if it's been pruned
```

Comment:
Where are we at with this, does the genesis get pruned? Seems like maybe we'd want to keep it around always.

Reply:
Yes, I need to look at a way to get the hash for this (when it's pruned) and as the comment above implies, this needs to be checked.h